bot/exts/moderation/stream.py

```diff
@@ -85,7 +87,7 @@ async def stream(self, ctx: commands.Context, member: discord.Member, duration:
         log.trace(f"Attempting to give temporary streaming permission to {member} ({member.id}).")
         # If duration is none then calculate default duration
         if duration is None:
-            duration = datetime.datetime.utcnow() + datetime.timedelta(
+            duration = arrow.utcnow().naive + timedelta(
```

Comment:
Why are you making it naive and then using `Arrow.fromdatetime` on it again to make it aware?

Reply:
Oh, I missed your Discord message. Anyway, you can avoid converting it back to aware by keeping it aware and only passing the naive version when the schedule function is called; adding the delta will still result in an `Arrow` object, right?

Reply:
Alternatively, `schedule_at` may be changed to somehow support both naïve and aware objects (probably by using now() in the object's timezone if it has one and utcnow otherwise).